public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -17,42 +17,42 @@ figure.image-display
   img(src="/resources/images/devguide/displaying-data/final.png" alt="Final UI")
 
 :marked
-  # Table Of Contents
+  # Contents
```

Comment:
Good question. I did a global search and found that in readme files we tend toward ## but in the jade files we use #. I'll keep an eye out for consistency.